### ISSUE TITLE
Does this still work on iOS 10?

### ISSUE THREAD

**Issue by KGeetings** [hackappcom/iloot#88]
Sorry if I'm just being dumb here, but as the title says. Does this work with backups on iOS 10? More specifically iOS 10.3.2

**Comment by Mocha-L** [comment]
I have the same Problem with u，and Get “Available Devices:  0” everytime. Do u Know why?

**Comment by FiZiX** [comment]
No, it hasn't worked since iOS 9. See this issue: https://github.com/hackappcom/iloot/issues/62
